README.md

```diff
@@ -267,6 +267,21 @@ Response:
 }
 ```
 
+### DELETE Endpoints:
```

Comment:
Nice work on this! I think we should change the watering endpoint from a post to a put or a patch as well

Reply:
I'll make a card for it and we can do that as a 'clean up' item.  Good catch!